<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sneak Archcompiler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="te_source">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_dest">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start_compile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_force">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Force recompile</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_copy_pyx">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy PYX files</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_copy_ini">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy INI files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save_preset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_kill">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>KILL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>140</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DESTINATION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>90</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SOURCE</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
